config/gazebo8_debian.upstream.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo8_debian_import
+method: http://packages.osrfoundation.org/gazebo/debian-stable
+suites: [stretch]
+component: main
+architectures: [amd64, arm64, armhf, i386, source]
+filter_formula: Package (% gazebo-dbg ) | Package (% gazebo ) | Package (% sdformat)
```

Comment:
You are right, we need the usual family of libraries. For `gazebo8`we have [this time](https://bitbucket.org/osrf/gazebo8-release/src/1bd521c4d1a19c6512b461fd1669e77a438ae8d1/ubuntu/debian/control?at=default&fileviewer=file-view-default#control-45):
```
     libsdformat5-dev,
     libignition-math3-dev,
     libignition-transport3-dev (>> 3.0.1)
     libignition-msgs-dev (>= 0.6.999)
```
I think that the rest of the dependencies are available from Ubuntu upstream.